monkestation/code/modules/machining/machining_recipes.dm

```diff
@@ -0,0 +1,977 @@
+//init functions
+/proc/init_machining_recipes()
+	. = list()
+	for(var/type in subtypesof(/datum/machining_recipe))
+		. += new type
+
+/proc/init_machining_recipes_atoms()
+	. = list()
+	for(var/datum/machining_recipe/recipe as anything in GLOB.machining_recipes)
+		// Result
+		. |= recipe.result
+		// Ingredients
+		for(var/atom/req_atom as anything in recipe.reqs)
+			. |= req_atom
+
+///Representative icons for the contents of each crafting recipe
+/datum/asset/spritesheet_batched/crafting/machining
+	name = "machining"
+
+/datum/asset/spritesheet_batched/crafting/machining/create_spritesheets()
+	var/id = 1
+	for(var/atom in GLOB.machining_recipes_atoms)
+		add_atom_icon(atom, id++)
+
+// base object for all machining recipes
+/datum/machining_recipe
+	/// in-game display name
+	/// Optional, if not set uses result name
+	var/name
+	/// description displayed in game
+	/// Optional, if not set uses result desc
+	var/desc
+	///type paths of items consumed associated with how many are needed
```

Comment:
Keep the autodoc consistent.
Either use a spacebar at the start for everything, or don't put a space for everything. Don't just flip flop

Also most of these could just be 1 single line. For multiline autodoc use 
```
	/**
	 * Autodoc multiline
	 */
```